.github/workflows/build-images.yml

```diff
@@ -93,6 +92,7 @@ jobs:
 
           - lang: dotnet10
             distro: debian
+            runner: macos-hosted
```

Comment:
Already tried moving dotnet10 to macos-hosted once and it was crashing for some reason. Will be nice to see how it performs now.